server/routes/visit.test.ts

```diff
@@ -211,41 +212,19 @@ describe('/visit/:reference', () => {
         })
     })
 
-    it('should render full booking summary page with visit information and prisoner tab selected, with default back link, formatting unknown contact telephone correctly', () => {
-      visitHistoryDetails.visit.visitContact.telephone = 'UNKNOWN'
```

Comment:
Checked with Dhiraj and we don't need to handle `UNKNOWN` as a special value for `telephone` so tidying this up.